### ISSUE TITLE
rdx2.lgtvsdp.com: what is it used for?

### ISSUE THREAD

**Issue by makruiten** [Perflyst/PiHoleBlocklist#64]
How do we know that this domain is actually used for ads/analytics/tracking? I can't find a single source on the internet saying what this domain is being used for. Is it only blocked because there are no ill side effects, or do we know what it is used for? Of course the same goes for some of the other domains, but this one stood out for me after investigating the LG domains. Most of the other domains either have the word "ad' in it, or have been proven to collect user data, distribute malware, etc.

**Comment by s1mwest** [comment]
Hey,, excuse me for chipping in here.  If you are using a blocklist for your smart TV, it's usually for the purpose of eliminating or minimising ads and other junk that manufacturers force on your screen.  A good rule of thumb; Block everything except what's necessary.
Provided that your TV is operating as you need it when using a blocklist then is it really important to know what the address is used for? :)
BTW, googling that address leads to hints that it or it's parent is used for data collection/telemetry.

**Comment by makruiten** [comment]
Thank you for you input. I decided to open this issue after I found out that two of the included domains were preventing the LG Store from functioning in my region. That made me question if the purpose of the blocked domains is clear in all cases. However, I do understand that a domain can have more than one function and it can also change in the future. That's a shortcoming of DNS level blocking: it's easy to have false positives, or collateral damage.

I do appreciate people sharing their blocklists and making them available in projects like this. However, for me proper functioning is more important than blocking one or two domains extra, and my current DNS level blocklist of 36,000 domains has caused me less issues than this small list. I think LG might be to blame here, as it seems to differ per region what domains can and can't be blocked. Anyway, I've kept rdx2.lgtvsdp.com on my blocklist and it hasn't caused any issues whatsoever, so I'm closing this issue.

**Comment by Perflyst** [comment]
> I decided to open this issue after I found out that two of the included domains were preventing the LG Store from functioning in my region.

Can you tell me which? They probably can be removed then.

> Anyway, I've kept rdx2.lgtvsdp.com on my blocklist and it hasn't caused any issues whatsoever,

Thanks, going to add this one.

**Comment by makruiten** [comment]
My region is NL, and these caused issues for me:

- nl.lgtvsdp.com
- ngfts.lge.com

I see that the latter has already been excluded in the main list, and the other one is the same issue as in GB: https://github.com/Perflyst/PiHoleBlocklist/issues/53. So these have actually been resolved already and I misremembered. The reason I misremembered is because the list that is included in NextDNS includes the main list with US domains only, so I had to add them by hand, and I had to exclude those. So false alarm, sorry! But at least another confirmation that those should really be excluded.
